mm2src/rpc/lp_commands/lp_commands.rs

```diff
@@ -35,3 +36,13 @@ pub async fn get_public_key(ctx: MmArc, _req: Json) -> GetPublicKeyRpcResult<Get
     let public_key = CryptoCtx::from_ctx(&ctx)?.secp256k1_pubkey().to_string();
     Ok(GetPublicKeyResponse { public_key })
 }
+
+#[derive(Serialize)]
+pub struct GetPublicKeyHashResponse {
+    public_key_hash: H160Json,
+}
+
+pub async fn get_public_key_hash(ctx: MmArc, _req: Json) -> GetPublicKeyRpcResult<GetPublicKeyHashResponse> {
+    let public_key_hash: H160Json = ctx.rmd160().to_owned().into();
```

Comment:
This is not a mistake but no need to explicitly annotate the type here, it will be inferred from the return type.